aries-site/src/examples/components/pageheader/PageHeaderAnatomy.js

```diff
@@ -1,9 +1,24 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import { Diagram, Grid, ResponsiveContext, Stack } from 'grommet';
+import { Diagram, Grid, ResponsiveContext, Stack, ThemeContext } from 'grommet';
 import { Annotation } from '../../../layouts';
 import { PageHeaderContentRegions } from '.';
 
+// need to overrid the PageHeader theme values in order for our anatomy
+// to fit correctly on the page. Even though it is using the correct breakpoints
+// our anatomy are only taking up half the space on the page so layout different than
+// a PageHeader on a full page.
+const PageHeaderSiteTheme = {
```

Comment:
```suggestion
const pageHeaderSiteTheme = {
```
small suggestion to align with conventions for capitalization